portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,23 +69,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       if(i < commentNum || commentNum > commentList.size()) {
         String commentEntity = (String) entity.getProperty("comments");
-        commentList.add(commentEntity);
+        String fullcomment = nickname + ": " + commentEntity + " ";
```

Comment:
nit: fullComment

Reply:
what does nit: mean?

Reply:
It refers to a nitpicky review comment. :)